CLOUDIFY_HOWTO.md

```diff
@@ -0,0 +1,171 @@
+# Deploy an existing Django application
+
+This guide will walk you through the steps need to deploy an existing
+Django application.
+
+## Before you begin
+
+Before running this guide, you must install the Cloud SDK, Cloud SQK Proxy.
+
+1. Download and install the
+   [Cloud SDK](https://cloud.google.com/sdk/docs/quickstarts)
+
+2. Download and install the
+   [Cloud SQL Proxy](https://cloud.google.com/sql/docs/mysql/connect-admin-proxy#install)
+
+3. Download and install [Docker](https://docs.docker.com/install/overview/)
+   (any edition)
+
+4. Prepare your environment for Python development. You have the Python 3.5 or
+   later, pip and virtualenv installed on your system. For instructions, refer
+   to the [Python Development Environment Setup Guide](https://cloud.google.com/python/setup).
+
+5. If you don't already one, you need to create a
```

Comment:
Do we assume they will accept the ToS by this step?

Reply:
I tried to create a new account and go to https://console.cloud.google.com/billing. The TOS automatically pops up after I visit this page.